README.md

```diff
@@ -1,2 +1,26 @@
-# tsdbcg
-A Deck Building Card Game built using X and Y working with A to Z.
+# Endpoints
+## Get `api/v1/gamestate/:game_id`
+
+Successfully returns a list of all cards as an array of objects. Empty values are returned as nil. in the format
+```
+{
+    name: "Copper",
+    category: "Money",
+    cost: 0,
+    victoryPoints: nil,
+    spendingPower: 1,
+    numberAvailable: 47
+    id: 1113
```

Comment:
I don't believe we are returning ID's at this time, because the Names are more important to the Frontend from our discussions.